packages/react-native-editor/__device-tests__/gutenberg-editor-audio.test.js

```diff
@@ -20,12 +19,7 @@ describe( 'Gutenberg Editor Audio Block tests', () => {
 		const block = await editorPage.getFirstBlockVisible();
 
 		block.click();
-
-		if ( isAndroid() ) {
-			await editorPage.driver.sleep( 5000 );
-		} else {
-			await editorPage.driver.sleep( 1000 );
-		}
+		await editorPage.driver.sleep( 5000 );
```

Comment:
In https://github.com/WordPress/gutenberg/pull/39377 we're introducing a new helper function to wait for an element to be visible, maybe we could use it here instead of waiting fixed time, wdyt?

**NOTE:** [The PR](https://github.com/WordPress/gutenberg/pull/39377) is not merged yet, although it's approved hence, you'd need to include those changes in this branch.

Reply:
Thanks for the headsup Carlos! Makes sense to wait for it to land. I will update the PR after that, to start using the `waitForVisible`, thanks!

Reply:
I [introduced](https://github.com/WordPress/gutenberg/pull/39381/commits/1e400525cfc424113968ab6ac84d4b0d1d107c01) a `waitForMediaLibrary` (wrapping around `waitForVisible` but for the specific media picker item) and [used that one in the tests](https://github.com/WordPress/gutenberg/pull/39381/commits/08149af3d728c1d87223e94842131a8266273e49), removing the `sleep`s. Let me know if you have any thoughts @fluiddot , ready for another pass!

Reply:
`waitForMediaLibrary` helper function looks great to me 👍 , I'll finish testing the changes locally and approve it.